### PR TITLE
Stylistic Improvements + Library Additions

### PR DESCRIPTION
**Pull request by ghost**
Hi, I understand if this PR is not accepted based on the fact that part of its changes falls outside what is covered in the [contributing guidelines](https://github.com/Siphalor/awesome-fabric/blob/master/CONTRIBUTING.md).

I just thought it would be cool (and harmless) to give improving it a shot!

Regardless of the outcome, have a wonderful day/night. 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,48 +1,167 @@
...
+
+- [Trinkets](https://github.com/emilyalexandra/trinkets) - A data-driven accessory mod that adds a slot group and slot system to Minecraft. ([Wiki](https://github.com/emilyploszaj/trinkets/wiki)) `MIT`
+
+### Items & Equippables
+
+- [Fabric Shield Lib](https://github.com/CrimsonDawn45/Fabric-Shield-Lib) - Library mod for easily adding shields, and shield enchantments into the game. ([Wiki](https://fabricmc.net/wiki/tutorial:shield)) `LGPL-2.1`
+
+### Java
+
 - [Fabric-ASM](https://github.com/Chocohead/Fabric-ASM) - Utilities for manipulating Java byte code and extending enums. `MPL-2.0`
```
Comment (Siphalor) [inline README.md]: Not sure about the category "Java" here.
Is it possible to use Fabric-ASM from Kotlin?

I'd suggest something like "Byte Code Manipulation" instead.

**Review by Siphalor — APPROVED** [review]
Well, you did a lot more than I could've hoped for :+1:

It is looking pretty good, and I have only one minor comment, that won't really block merging.

Thanks for reviving this project! :D

**Comment by ghost** [comment]
I'm so glad you like the work I've done! I've made the necessary edit, should be good to bring in now!

**Comment by Siphalor** [comment]
Let's go ;)

**Pull request merged**